<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Probability_Algorithm</class>
 <widget class="QWidget" name="Probability_Algorithm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
margin: 0;
background-color: rgb(40, 40, 40);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QFrame" name="gridFrame">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="gridFrame">
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">/* Cube size: */


color: #969696;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>Half-axis a</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="axisALineEdit">
           <property name="styleSheet">
            <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QFrame" name="gridFrame_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">
color: #969696;
font-family: Inter;
font-size: 13px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>If a sphere is used for construction - all halfi-axes must be equal.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="gridFrame_4">
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">/* Cube size: */


color: #969696;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>Orintation angle (A)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="orintationAngleLineEdit">
           <property name="styleSheet">
            <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="gridFrame_3">
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Cube size: */


color: #969696;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>Half-axis b</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="axisCLineEdit">
           <property name="styleSheet">
            <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="gridFrame_2">
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>224</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Cube size: */


color: #969696;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>Half-axis c</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="axisBLineEdit">
           <property name="styleSheet">
            <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="gridFrame_6">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Cube size: */


color: #969696;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>Orientation angle (B)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="gridFrame_7">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">/* Cube size: */


color: #969696;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
           </property>
           <property name="text">
            <string>Orientation angle (C)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="styleSheet">
            <string notr="true">background: #282828;
border: 1px solid #969696;
border-radius: 5px;
color: #969696;
padding: 4px 0;
padding-left: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="gridFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="cancelPushButton">
        <property name="styleSheet">
         <string notr="true">padding: 15px;

border-radius: 15px;
border: 1px solid #969696;
background: #282828;

color: rgba(150, 150, 150, 0.50);
text-align: center;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;
text-transform: uppercase;
margin: 0 5px;</string>
        </property>
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="applyPushButton">
        <property name="styleSheet">
         <string notr="true">padding: 15px;

border-radius: 15px;
border: 1px solid #969696;
background: #282828;

color: rgba(150, 150, 150, 0.50);
text-align: center;
font-family: Inter;
font-size: 20px;
font-style: normal;
font-weight: 700;
line-height: normal;
text-transform: uppercase;
margin: 0 5px;</string>
        </property>
        <property name="text">
         <string>APPLY</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
